staging.json

```diff
@@ -376,26 +606,152 @@
         "proxy": "0xdaaa8bad96512dcef7aaf7af98d2ef14f6117cd7"
       },
       "replicas": {
+        "arbitrum-rinkeby": {
+          "beacon": "0xd8be3c0d3ce1c077184a931cc88f938ec5e639de",
+          "implementation": "0xa179a65273b55a424ddb2834ce3983f4b5d6a5fb",
+          "proxy": "0x0499063d42b5d3186e8bcd9d7d84e9c6d4efaa6c"
+        },
+        "bsc-testnet": {
+          "beacon": "0xd8be3c0d3ce1c077184a931cc88f938ec5e639de",
+          "implementation": "0xa179a65273b55a424ddb2834ce3983f4b5d6a5fb",
+          "proxy": "0x32641eb9af76a6d62e44d09e7a7f02f6ff5bc5d7"
+        },
         "evmostestnet": {
-          "beacon": "0xd8Be3C0d3CE1C077184a931cc88F938Ec5E639dE",
-          "implementation": "0xA179A65273b55A424DDB2834ce3983f4b5d6A5Fb",
-          "proxy": "0x41737AC8A129F20707D914ff088a7eC3e57bC99B"
+          "beacon": "0xd8be3c0d3ce1c077184a931cc88f938ec5e639de",
```

Comment:
note to self: why did it overwrite some existing replicas?

how will that affect the system?

Reply:
it was actually just different case of the same address, all good